src/systems/thruster/Thruster.cc

```diff
@@ -362,10 +396,19 @@ void Thruster::Configure(
 
     feedbackTopic = gz::transport::TopicUtils::AsValidTopic(
         "/model/" + ns + "/joint/" + jointName + "/force");
+
+    this->dataPtr->deadbandTopic = gz::transport::TopicUtils::AsValidTopic(
+      "/model/" + ns + "/joint/" + jointName + "/enable_deadband");
   }
 
   gzmsg << "Thruster listening to commands on [" << thrusterTopic << "]"
         << std::endl;
+  this->dataPtr->node.Subscribe(
+      this->dataPtr->deadbandTopic,
```

Comment:
If the `deadbandTopic` is empty, we should not subscribe. This does happen during the tests and we get a complaint from `gz-transport`

Reply:
The complaints do not come from the plugin but from the tests them-selves.
In the test I create a publisher to `deadbandTopic` which is empty in most unit test except the one I added. So I added a check for emptiness there.
I also did not subscribe to the topic in the plugin, if the topic is empty, which is also a good idea.

Reply:
Thanks! I didn't notice that.